salt/states/pkg.py

```diff
@@ -1448,6 +1482,14 @@ def installed(
 
     kwargs['saltenv'] = __env__
     refresh = salt.utils.pkg.check_refresh(__opts__, refresh)
+
+    # check if capabilities should be checked and modify the requested packages
+    # accordingly.
+    if pkgs:
+        (pkgs, was_refreshed) = _resolve_capabilities(pkgs, refresh=refresh, **kwargs)
```

Comment:
Extra parenthesis. PyLint should be mad by now. :wink:

Reply:
will be fixed everywhere.